src/metabase/sync/analyze/fingerprint.clj

```diff
@@ -212,18 +212,14 @@
     log-progress-fn :- LogProgressFn]
    (fingerprint-fields-for-db!* database tables log-progress-fn (constantly true)))
 
-  ;; TODO: Maybe the driver should have a function to tell you if it supports fingerprinting?
   ([database        :- i/DatabaseInstance
     tables          :- [:maybe [:sequential i/TableInstance]]
     log-progress-fn :- LogProgressFn
     continue?       :- [:=> [:cat ::FingerprintStats] :any]]
    (qp.store/with-metadata-provider (u/the-id database)
      (reduce (fn [acc table]
                (log-progress-fn (if *refingerprint?* "refingerprint-fields" "fingerprint-fields") table)
-               (let [results (if (= :googleanalytics (:engine database))
-                               (empty-stats-map 0)
```

Comment:
Wow, it was unfortunate we did so much work to accumulate these zeros previously! 😅